prepare_setup.sh

```diff
@@ -252,6 +250,26 @@ install_helm(){
     ln -s /usr/local/linux-amd64/helm /usr/local/bin/helm
 }
 
+packages_install(){
+    local distro=$(get_distro)
+    local_status=0
+    packages="conntrack"
```

Comment:
why you need conntrack?

Reply:
It is needed for kubernetes

Reply:
from the logs `08:39:02 	[ERROR FileExisting-conntrack]: conntrack not found in system path`